SampleApps/TestApp/AppDelegate.swift

```diff
@@ -10,28 +10,36 @@
 // governing permissions and limitations under the License.
 //
 
-import AEPAssurance
 import AEPCore
-import AEPEdge
-import AEPEdgeConsent
 import AEPEdgeIdentity
 import AEPServices
 import Compression
 import UIKit
 
+// MARK: TODO remove this once Assurance has tvOS support.
+#if os(iOS)
+import AEPAssurance
+import AEPEdge
+import AEPEdgeConsent
+#endif
+
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
     private let ENVIRONMENT_FILE_ID = ""
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        var extensions: [NSObject.Type] = [Identity.self]
+        // MARK: TODO remove this once Assurance has tvOS support.
+        #if os(iOS)
+        extensions.append(Assurance.self)
+        extensions.append(Consent.self)
+        extensions.append(Edge.self)
```

Comment:
So for tvOS there's no support for Edge and Consent? Is that just because we haven't finished that work? Will we need to come back and update the sample app once Edge and Consent support tvOS?

Reply:
Yes, we will have to update this once we Edge and Consent have tvOS support released.